<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/bleu.png);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_nom_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nbrp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifier">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sexe_">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sexe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_motp_">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>motp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
